src/content/1.7/modules/concepts/controllers/admin-controllers/_index.md

```diff
@@ -35,6 +49,16 @@ class DemoController extends FrameworkBundleAdminController
 You have access to the Container, to Twig as rendering engine, the Doctrine ORM, everything from Symfony framework ecosystem.
 Note that you must return a `Response` object, but this can be a `JsonResponse` if you plan to make a single point application (or "SPA").
 
+In order to use symfony DI you need to declare some yml config:
```

Comment:
```suggestion
If you want Symfony Dependency Injection to inject services into your controller, you need to use specific YAML service declaration:
```

Also, I think it only works since 1.7.7 ? 🤔 might be worth it to mention it

Reply:
just tested with @pablopolyte it seems to be working fine on 1.7.6

Reply:
So it's unrelated to "controllers as services" 😄 good news !